src/Interface/Tabs/PluginList.as

```diff
@@ -136,8 +144,9 @@ class PluginListTab : Tab
 	{
 		CheckRequest();
 
-		if (m_request !is null) {
+		if (m_request !is null && m_pageCount == 0) {
 			UI::Text("Loading list..");
+			m_lastPageRequestFinished = Time::Now;
```

Comment:
Why is this here? It seems like it's not the intended purpose considering the variable name. Is there a different solution to accomplish the same thing, in a cleaner and more obvious way?

Reply:
Some leftover code to ensure that the 500ms debounce doesn't start till the first frame that the table is drawn.
That ends up happening in the expected place, though, too. I removed it and there's no apparent difference.